### DISCUSSION TITLE
What's the status of this project

### DISCUSSION THREAD

**Discussion by HolidayLighting** [mRemoteNG/mRemoteNG#2635] [General]
I've been using this for a long time and just built a new PC and went to install this and found some issues that I don't have with the version on my old PC.

1. Old PC and New PC When I check for an update it says I have a version newer than what's available.  How is that?
2. The Dark Themes don't do a great job, making everything dark and harder to read.
3. On a large screen monitor the fonts are all very small without any way to make things bigger.

**Comment by Kvarkas** [comment]
1. ignore that for a while, due another versioning mechanism older versions provide incorrect information for update, only manual update is suggested, NB current version have disabled save settings mechanism due be able to run together with other versions and dont mess them up. 
2. Theming is bad at this moment, need separate work what planed to start on next release perhaps. 
3. That know issue but due .net 6 limitations for winforms imposible to fix, expected better results after migration to .net8 on next release.
